salt/orchestrate/aws/rds.sls

```diff
@@ -57,14 +61,35 @@ set_{{ name }}_master_password_in_vault:
 {% set master_pass = master_pass.data.value %}
 {% endif %}
 
+{# If engine is postgres and version is 10, then use major version only for
+   parameter group family. All others use major.minor. #}
+{% if engine == 'postgres' and int(engine_version.partition('.')[0]) > 9 %}
+{% set fam_version = engine_version.partition('.')[0] %}
+{% else %}
+{% set fam_version = engine_version.rpartition('.')[0] %}
+{% endif %}
+{% set db_parameter_group_family = engine ~ fam_version %}
+{% set db_parameter_group_name = ENVIRONMENT ~ '_' ~ engine ~ '_' ~ name ~ '_parameters' %}
+
+{% if parameters %}
```

Comment:
We always want the parameter group to be created. If there needs to be a conditional it should only apply to setting the customized parameters on the parameter group (line 79). This also means unconditionally applying the parameter group to the RDS instance on line 91.

Reply:
I've changed this in commit ace521b.